packages/application/src/submission.tsx

```diff
@@ -176,10 +177,25 @@ export class SubmissionHandler {
           dialogTitle = 'Job submission to ' + runtime_config + ' succeeded';
           dialogBody = (
             <p>
-              Check the status of your run at{' '}
-              <a href={data.url} target="_blank" rel="noopener noreferrer">
+              Check the status of your experiment at{' '}
+              <a
+                href={data['experiment-url']}
+                target="_blank"
+                rel="noopener noreferrer"
+              >
                 Run Details
               </a>
+              <br />
+              Check the experiment results and outputs at{' '}
+              <a
+                href={data['object-storage-url']}
+                target="_blank"
+                rel="noopener noreferrer"
+              >
+                object storage.
+              </a>
+              <br />
+              Located in the {data['object-storage-path']} working directory
```

Comment:
This statement feels like part of the previous sentence since its not a complete sentence.  Seems like it should be...

_Check the experiment results and outputs at {object-storage-url} object storage and located in the {object-storage-path} working directory._

or 

_Check the experiment results and outputs at {object-storage-url} object storage.
These files are located in the {object-storage-path} working directory._

Or does the 'Located in the ...' statement refer to a _different_ set of files entirely?

Reply:
The new dialog looks like this now:

![image](https://user-images.githubusercontent.com/382917/82848764-d1846d80-9ea9-11ea-9388-edfef6d67434.png)